app/features/welcome/styled/Header.js

```diff
@@ -6,5 +6,5 @@ export default styled.div`
     align-items: center;
     display: flex;
     margin: 0 auto;
-    padding: 30px;
+    padding: 8em;
```

Comment:
I was using `px` because we didn't have much text before.
But since we have a little text I thought let's switch to `em`. I'll change all of them to `em`.

Reply:
Thanks!